### ISSUE TITLE
Failing test in "Fair's Extramarital Affairs Data"

### ISSUE THREAD

**Issue by bkmgit** [red-data-tools/red-datasets#111]

**Comment by kou** [comment]
Could you write failure details?

**Comment by bkmgit** [comment]
Record name changed to 1745 so comparison test failed.

Other issue is a parsing error because https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/multilabel.html is not valid xml

**Comment by bkmgit** [comment]
Parsing error output
```
===============================================================================
Error: test: multi-label(LIBSVMDatasetTest):
  REXML::ParseException: malformed XML: missing tag start
  Line: -1
  Position: -1
  Last 80 unconsumed characters:
  <', '"' and '&' are replaced with their textual representatons 'gt', 'lt', 'quot'
/opt/hostedtoolcache/Ruby/2.6.8/x64/lib/ruby/gems/2.6.0/gems/rexml-3.2.5/lib/rexml/parsers/baseparser.rb:393:in `pull_event'
/opt/hostedtoolcache/Ruby/2.6.8/x64/lib/ruby/gems/2.6.0/gems/rexml-3.2.5/lib/rexml/parsers/baseparser.rb:183:in `pull'
/opt/hostedtoolcache/Ruby/2.6.8/x64/lib/ruby/gems/2.6.0/gems/rexml-3.2.5/lib/rexml/parsers/streamparser.rb:20:in `parse'
/home/runner/work/red-datasets/red-datasets/lib/datasets/libsvm-dataset-list.rb:92:in `block (2 levels) in parse_detail'
/home/runner/work/red-datasets/red-datasets/lib/datasets/libsvm-dataset-list.rb:89:in `catch'
/home/runner/work/red-datasets/red-datasets/lib/datasets/libsvm-dataset-list.rb:89:in `block in parse_detail'
/home/runner/work/red-datasets/red-datasets/lib/datasets/libsvm-dataset-list.rb:82:in `block in open_detail'
/home/runner/work/red-datasets/red-datasets/lib/datasets/libsvm-dataset-list.rb:81:in `open'
/home/runner/work/red-datasets/red-datasets/lib/datasets/libsvm-dataset-list.rb:81:in `open'
/home/runner/work/red-datasets/red-datasets/lib/datasets/libsvm-dataset-list.rb:81:in `open_detail'
/home/runner/work/red-datasets/red-datasets/lib/datasets/libsvm-dataset-list.rb:88:in `parse_detail'
/home/runner/work/red-datasets/red-datasets/lib/datasets/libsvm-dataset-list.rb:43:in `block (3 levels) in each'
/home/runner/work/red-datasets/red-datasets/lib/datasets/libsvm-dataset-list.rb:129:in `tag_end'
/opt/hostedtoolcache/Ruby/2.6.8/x64/lib/ruby/gems/2.6.0/gems/rexml-3.2.5/lib/rexml/parsers/streamparser.rb:36:in `parse'
/home/runner/work/red-datasets/red-datasets/lib/datasets/libsvm-dataset-list.rb:47:in `block (2 levels) in each'
/home/runner/work/red-datasets/red-datasets/lib/datasets/libsvm-dataset-list.rb:41:in `catch'
/home/runner/work/red-datasets/red-datasets/lib/datasets/libsvm-dataset-list.rb:41:in `block in each'
/home/runner/work/red-datasets/red-datasets/lib/datasets/libsvm-dataset-list.rb:57:in `block in open_data'
/home/runner/work/red-datasets/red-datasets/lib/datasets/libsvm-dataset-list.rb:56:in `open'
/home/runner/work/red-datasets/red-datasets/lib/datasets/libsvm-dataset-list.rb:56:in `open'
/home/runner/work/red-datasets/red-datasets/lib/datasets/libsvm-dataset-list.rb:56:in `open_data'
/home/runner/work/red-datasets/red-datasets/lib/datasets/libsvm-dataset-list.rb:40:in `each'
/home/runner/work/red-datasets/red-datasets/lib/datasets/libsvm.rb:70:in `fetch_dataset_info'
/home/runner/work/red-datasets/red-datasets/lib/datasets/libsvm.rb:38:in `initialize'
/home/runner/work/red-datasets/red-datasets/test/test-libsvm.rb:68:in `new'
/home/runner/work/red-datasets/red-datasets/test/test-libsvm.rb:68:in `block in <class:LIBSVMDatasetTest>'
===============================================================================
```

**Comment by kou** [comment]
Fixed by #112.
